npt/util.py

```diff
@@ -543,16 +545,21 @@ def parse_cmdline() -> OptionContainer:
              f"draft format : draft[-rev][.extn]."
              f"rfc format : rfc[.extn]")
 
-    _obj = ap.parse_args()
+    _obj = ap.parse_args(arglist)
+
     opt = OptionContainer(pathlib.Path(_obj.dir[0]),
                           DownloadOptions(force=_obj.force),
                           _obj.outformat[0].split(sep=','), [])
+    return (_obj, opt )
+
+
 
-    if _obj.newdraft:
-        fromdate = datetime.strptime(_obj.newdraft, "%Y-%m-%d %H:%M:%S")
+def setup_opts( cmd_obj: argparse.Namespace , opt: OptionContainer) -> OptionContainer :
+    if cmd_obj.newdraft:
+        fromdate = datetime.strptime(cmd_obj.newdraft, "%Y-%m-%d %H:%M:%S")
```

Comment:
`fromdate = datetime.fromisoformat(cmd_obj.newdraft)`?

Reply:
It was a general comment for all the uses of `strptime()`, not just this one.

Reply:
I believe I had done all instances of strptime().  Have I missed any ? Just grep()d  through the directory and could not find any calls to strptime().

Reply:
Sure, but I see 7 uses of `strftime()`

Reply:
Done ! Uses isotime() API instead of strptime()  and strftime() .